src/routes/search/containers/SearchModalContainer.js

```diff
@@ -39,8 +37,8 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType) => {
     language: state.contentLanguage,
     cityCode: city,
     closeModal: () => { ownProps.navigation.goBack() },
-    sendFeedback: async (comment: string, query: string) => {
-      await feedbackEndpoint.request({
+    sendFeedback: async (comment: string, query: string, apiUrlOverride: ?string = undefined) => {
+      await createFeedbackEndpoint(apiUrlOverride || baseUrl).request({
```

Comment:
Also shouldn't `apiUrlOverride || baseUrl` be  `baseUrl || apiUrlOverride`

Reply:
Nope, you want to use the override `apiUrlOverride` (if it exists) and only use `baseUrl` as default fallback.